.gitignore

```diff
@@ -143,3 +143,6 @@ dmypy.json
 
 # Mac OS
 .DS_Store
+
+# IntelliJ IDE idea folder
+.idea/
```

Comment:
I think this diff should be reverted

Reply:
@Arka161 I tried to find a solution to exclude the ideas folder from pycharm but wasnt able to find something convincing. Do you have a solution in mind?

Reply:
@johntremblay I basically meant something like avoiding any new lines in the `.gitignore`, and checking out this file from `develop` as it's just an IDE change, and not essential for speechbrain. Though it's just a clean coding convention.